src/test/fixtures/sample-config-ropsten-aws.json

```diff
@@ -18,10 +19,10 @@
         "issuers": [
           {
             "name": "Demo Issuer",
-            "documentStore": "0x6e80FD3dD1151607Ce15B98eE1e3DDd4E4DdF0B6",
+            "documentStore": "0xAa2BaF74bD975526025Ae5EfAF08798Dd2856107",
             "identityProof": {
               "type": "DNS-TXT",
-              "location": "example.openattestation.com"
+              "location": "siachenren.mywork.business"
```

Comment:
🤨

Reply:
ops sorry, this is from my personal aws testing key, will remove any mention :x